specifyweb/stats/views.py

```diff
@@ -72,24 +73,10 @@ def collection_preparations(request) -> HttpResponse:
             }
         }
     }}, )
-def collection_locality_geography(request) -> HttpResponse:
-    cursor = connection.cursor()
-    ####LOC_GEO
-    # COUNTRIES
+def collection_locality_geography(request, stat) -> HttpResponse:
     geography_dict = {}
-    #don't need geographyid
-    cursor.execute(
-        """
-        SELECT count(Name)
-        FROM (SELECT DISTINCT Name
-              FROM (SELECT g.GeographyID, g.nodenumber
-                    FROM locality as l
-                             inner join geography as g on l.GeographyID = g.GeographyID) as GEO,
-                   geography as g
-              WHERE g.rankid = 200
-                and GEO.nodenumber between g.nodenumber and g.HighestChildNodeNumber) As GEO2
-        """)
-    geography_dict['countries'] = int((cursor.fetchone()[0]))
+    if stat == 'percentGeoReferenced':
```

Comment:
would it make sense to return 404 here instead of empty dictionary if stat is not percentGeoReferenced?

Reply:
we can - frontend will crash (which is fair, don't make invalid calls!)  I'll raise 404!